authserver/db/models/models.py

```diff
@@ -26,60 +26,16 @@
                  db.Column('role_id', db.String, db.ForeignKey('oauth2_roles.id'), primary_key=True))
 
 
-class Organization(db.Model):
-    """Data Trust Organization."""
-    __tablename__ = 'organizations'
-
-    id = db.Column(db.String, primary_key=True)
-    name = db.Column(db.String(40), unique=True, nullable=False)
-    url = db.Column(db.String)
-    date_created = db.Column(db.TIMESTAMP)
-    date_last_updated = db.Column(db.TIMESTAMP)
-
-    def __init__(self, name, url=None):
-        self.id = str(uuid4()).replace('-', '')
-        self.name = name
-        self.url = url
-        self.date_created = datetime.utcnow()
-        self.date_last_updated = datetime.utcnow()
-
-    def __str__(self):
-        return '{} - {}'.format(self.id, self.name)
-
-
-class OrganizationSchema(ma.SQLAlchemySchema):
-    """Organization Schema
-
-    A marshmallow schema for validating the Organization model.
-    """
-
-    class Meta:
-        ordered = True
-        model = Organization
-
-    id = ma.auto_field(dump_only=True)
-    name = ma.auto_field(required=True)
-    url = ma.auto_field()
-    date_created = ma.auto_field(dump_only=True)
-    date_last_updated = ma.auto_field(dump_only=True)
-
-
 class User(db.Model):
     """Data Trust User."""
     __tablename__ = 'users'
     __table_args__ = (db.UniqueConstraint('username'), )
 
     id = db.Column(db.String, primary_key=True)
+    person_id = db.Column(db.String)
     username = db.Column(db.String(40), unique=True, nullable=False)
-    firstname = db.Column(db.String(40), nullable=False)
-    lastname = db.Column(db.String(40), nullable=False)
-    organization = db.relationship(
-        'Organization', backref='users', lazy='subquery')
-    organization_id = db.Column(
-        db.String, db.ForeignKey('organizations.id', ondelete='CASCADE'), nullable=False)
-    email_address = db.Column(db.String(40), nullable=False)
-    telephone = db.Column(db.String(20), nullable=True)
-    active = db.Column(db.Boolean, nullable=False, default=True)
+    active = db.Column(db.Boolean, nullable=False, default=False)
+    can_login = db.Column(db.Boolean, nullable=True, default=False)
```

Comment:
Two thoughts:

1. I think we want this field to be `nullable=False`, if we want to set a default value. 
2. Will the migrations add the default value to existing rows?

Reply:
@reginafcompton interestingly enough, 1. and 2. are related. I set nullable to true to prevent legacy things from breaking. I also pushed an update that will check if `can_login` is null and return false if it turns out to be.

Reply:
Ah, is there a way to have the migrations populate existing rows (i.e., the User entries that do not have `can_login`)?

Reply:
There is, but I don't think it's worth the effort right now and I'd honestly have an ETL handle these particular tasks because it won't always be possible to force a set of containers to restart all the time.